### DISCUSSION TITLE
Load Queries

### DISCUSSION THREAD

**Discussion by jrschenk** [noahgorstein/jqp#48] [Ideas]
This is a really good tool. I just found out about it from a YouTube video by DevOps Toolbox. 
I think it would be useful to be able to load a set of queries from a file when the program starts up.

**Comment by noahgorstein** [comment]
Can you share some more about this? jqp does take a file as input. What would loading multiple queries look like in your ideal world?

**Reply by jrschenk** [reply]
I notice that jqp maintains a history of the queries while the running the program. As a start, a mechanism to populate this history on startup. Taking it a step further, a means of naming the queries would be useful and the capabilty to select a query based on its name. I think this would fit into the YAML config . 

I have only used jqp a couple of times but it was quite useful. I was using jqp on json files that always have the same structure, and I ended up saving my queries to a text file that I could cut and paste into jqp the next time I ran the program.
